tests/Feature/Api/TracksApiTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use App\Track;
+use App\Album;
+use App\Reciter;
+use App\User;
+use Faker;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Laravel\Passport\Passport;
+use Tests\TestCase;
+
+class TracksApiTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * A basic test example.
+     *
+     * @return void
+     */
+    public function testGetTrackById()
+    {
+        /** @var User $user */
+        /** @var Reciter $reciter */
+        /** @var Album $album */
+        /** @var Track $track */
+        $user = factory(User::class)->create();
+        $reciter = factory(Reciter::class)->create(['created_by' => $user->id]);
+        $album = factory(Album::class)->create(['created_by' => $user->id, 'reciter_id' => $reciter->id]);
+        $track = factory(Track::class)->create(['created_by' => $user->id, 'reciter_id' => $reciter->id, 'album_id' => $album->id]);
```

Comment:
Some spacing here between the creation of models and the test run would be appreciated.